### ISSUE TITLE
CacheMonkey Doesn't Work/Scan anything on Windows 11

### ISSUE THREAD

**Issue by harnandodisoto** [jamiepine/cache-monkey#72]
![screenshot 2](https://github.com/jamiepine/cache-monkey/assets/5673900/014f3d2d-7266-471e-a6f7-b187b5f2576c)

So I have a normal instance of Chrome/Discord and have it set to the default file directory yet cachemonkey doesn't appear to see anything at all when scanning for a millisecond despite the cache_data file having multiple data files inside them. I do not know what's wrong it doesn't seem to change the result even with a reinstall of Cachemonkey or give it admin privileges.

**Comment by jamiepine** [comment]
Yeah I haven't touched this in years so it's probably not gonna work. My best advice if you need the cache files is to use a command to turn them into pngs, ask chatgpt for one cause I can't remember it.

**Comment by jamiepine** [comment]
`ren *.* *.png` should do it. run it inside the folder with the cache files.

**Comment by harnandodisoto** [comment]
Well, frankly I'm not too tech-savvy so that's going to be a fun journey to figure out. I assume this is a cmd prompt command or something that has to be posted through the Linux terminal?

> `ren *.* *.png` should do it. run it inside the folder with the cache files.

**Comment by AlekEagle** [comment]
Honestly Jamie, I think it might be time cache monkey gets archived.

**Comment by AlekEagle** [comment]
> Well, frankly I'm not too tech-savvy so that's going to be a fun journey to figure out. I assume this is a cmd prompt command or something that has to be posted through the Linux terminal?
> 
> > `ren *.* *.png` should do it. run it inside the folder with the cache files.

Yes, you would run that in your command prompt in the cache directory for whatever chromium based program you are trying to view the cache for.

**Comment by AXOca** [comment]
If you ever come back to this issue thread, just do following:

[CacheMonkey]
- Settings -> 
- Navigate to "Application Cache Folders" ->
- Click the folder icon next to "Cache/PATH" and choose discord/Cache/Cache_data
Done.

This worked for me.
